operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -227,9 +227,12 @@ function buildWDTParams_MD5() {
   fi
 
   if [ "${WDT_DOMAIN_TYPE}" == "JRF" ] && [ ! -f "${OPSS_KEY_PASSPHRASE}" ] ; then
-    trace SEVERE "Domain Source Type is 'FromModel' and domain type JRF which requires specifying a " \
-       "walletPasswordSecret in your domain resource and deploying this secret with a 'walletPassword' key, " \
-       " but the secret does not have this key."
+    trace SEVERE "The domain resource 'spec.domainHomeSourceType'" \
+       "is 'FromModel' and the 'spec.configuration.model.domainType' is 'JRF';" \
+       "this combination requires specifying a" \
+       "'spec.configuration.model.walletPasswordSecret' in your domain" \
+       "resource and deploying this secret with a 'walletPassword' key," \
+       "but the secret does not have this key."
     exit 1
```

Comment:
it should be exitOrLoop for consistency

Reply:
done, plus i made the same change to another 'exit 1' in the same file